src/ConstAddressDeployer.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+contract ConstAddressDeployer {
+    event Deployed(bytes32 bytecodeHash, bytes32 salt, address deployedAddress);
+    function deploy(bytes memory bytecode, bytes32 salt) external returns (address deployedAddress) {
```

Comment:
Can bytecode reside in calldata here or it won't work with mload below?

Reply:
I think it needs to be memory.